lib/Slack_web_api.js

```diff
@@ -127,10 +127,7 @@ module.exports = function(bot, config) {
      * @param {function} cb A NodeJS-style callback
      */
     slack_api.callAPI = function(command, data, cb, multipart) {
-        if (!data.token) {
-            data.token = config.token;
-        }
-
+        data.token = data.token ? data.token : config.token;
```

Comment:
The problem with the || operator is that if the var is not defined it will throw an error.
Example:
testFunction(i) {
i = i || 10; //PERFECT
let t = t || 10; //ERROR because t is not defined
}